assets/js/dependencies/async_loader.js

```diff
@@ -256,3 +274,17 @@ _rStudioRequest=function(url,method,shared_secret,port,params){
     }
   });
 };
+
+// Read a page's GET URL variables and return them as an associative array.
+getUrlVars = function()
```

Comment:
No, urlParam did not found this parameter. Might be because I do not add it when requesting the page, but only to view it in the rstudio-browser. I could look to solve it if you want, would be cleaner.